journal/202012.md

```diff
@@ -0,0 +1,320 @@
+# {DRAFT} Decred Journal – December 2020
+
+![{alt text}](../img/{file.jpg} "{hover tooltip text}")
+
+_Image: {"$title by $author. $optional-description" for artwork, "$description" for photos}_
+
+{ introduction, major news of the month ordered by impact, don't put any links here }
+
+## Development
+
+Unless otherwise noted, the work reported here has the "merged to master" status. It means that the work is completed, reviewed, and integrated into the source code that advanced users can build and run, but is not yet available in release binaries for regular users.
+
+**[dcrd](https://github.com/decred/dcrd)**
+
+Merged in master and backported to v1.6 release:
+
+- corrected [fee calculations](https://github.com/decred/dcrd/pull/2530) to fix some transactions being considered lower priority in some reorg scenarios
+- whitelisted DCP-5 [violations](https://github.com/decred/dcrd/pull/2533). Due to a bug that has since been fixed, 5 blocks were accepted with an old block version 6 when the majority of the network had already upgraded to version 7. The blocks were otherwise valid and are now whitelisted to allow fully syncing the chain without checkpoints. { is this wording good enough? is no-checkpoint sync the primary reason for whitelisting? how "bad" was the bug and which PR fixed it? }
+- { does the extra commit to use [blockchain v3.0.3](https://github.com/decred/dcrd/pull/2539) add anything to v1.6 on top of the above two changes? if not, why this PR is necessary when previous commits have already ported the fixes? (sorry not well versed with Go module system) }
```

Comment:
Full software releases only use release modules, so the commit necessary to pull the changes in.  Thus, the individual fixes don't actually need to be ported to the release branch.  It does have a slight benefit in making it more clear exactly what the new version pulled in though.